Monika After Story/game/script-stories.rpy

```diff
@@ -348,11 +412,556 @@ label mas_story_immortal_love:
+    show emptydesk at i11 zorder 9
+    m "It is instead on the figure that now looms above him as a former vision of the one he once loved."
+    # 1 in 10
+    if renpy.random.randint(1,10) == 1 or mas_full_scares:
+        hide monika
+        play sound "sfx/giggle.ogg"
+        show yuri dragon2 zorder 15 at malpha
+        $ style.say_dialogue = style.edited
+        y "{cps=*2}I'll get you too.{/cps}{nw}"
+        hide yuri
+        $ style.say_dialogue = style.normal
+        show monika 1eua at i11 zorder MAS_MONIKA_Z
+    m "The next morning, the villagers found the hunter dead at the edge of the forest."
+    hide emptydesk
+    call mas_scary_story_cleanup
+    return
+
+init 5 python:
+    addEvent(Event(persistent._mas_story_database,eventlabel="mas_scary_story_kuchisake_onna",
+    category=[store.mas_stories.TYPE_SCARY], prompt="Kuchisake-Onna",unlocked=False),
+    eventdb=store.mas_stories.story_database)
+
+label mas_scary_story_kuchisake_onna:
+    call mas_scary_story_setup
+    m 3eua "There once was a beautiful woman, whom was the wife of a samurai."
+    m 3eub "Not only was she very beautiful, but she was very vain, and welcomed the attention of any man prepared to offer it to her."
+    m 1tsu "She would often ask men to rate her appearance."
+    m 1esd "The woman ended up cheating on her husband and he quickly found out."
+    m 1efd "When he confronted the woman about it, they got into an argument and the samurai ended up slitting her mouth from ear to ear, in the shape of a wide smile."
+    m "Then he asked her, 'who will think you are beautiful, now?'."
+    m 2dsd "Shortly after, the woman died."
+    m "The samurai, full of regret for what he had done, committed seppuku shortly afterwards as well."
+    m 3eud "The woman’s story doesn’t end here though, she came back as a vengeful spirit."
+    m "They say she now wanders around aimlessly at night, her face covered with a surgical mask."
+    m 1esa "When she comes across someone walking by themselves, she will suddenly pose the question, ‘am I pretty?’."
+    # 1 in 15
+    if renpy.random.randint(1,15) == 1 or mas_full_scares:
+        hide monika
+        show screen tear(20, 0.1, 0.1, 0, 40)
+        play sound "sfx/s_kill_glitch1.ogg"
+        show natsuki ghost2 zorder 16 at i11
+        show k_rects_eyes1 zorder 17
+        show k_rects_eyes2 zorder 17
+        $ pause(0.25)
+        #play music t7g
+        stop sound
+        hide screen tear
+        $ style.say_dialogue = style.edited
+        show screen mas_background_timed_jump(5, "mas_scary_story_kuchisake_onna.no")
+        menu:
+            "am I pretty?"
+            "Yes":
+                hide screen mas_background_timed_jump
+                jump mas_scary_story_kuchisake_onna.clean
+            "No":
+                jump mas_scary_story_kuchisake_onna.no
+    else:
+        jump mas_scary_story_kuchisake_onna.end
+
+    label .no:

```

Comment:
so we had issues with local labels in the chess cheating dialogue flow. My understanding was that the label needed to be reached first before we could reference it in jumps/calls. Is that not the case?

Note that this kind of issue only surfaces when using a release version, not via renpy sdk.

Reply:
I'll test on a release version and change accordingly to what my tests show